.cirrus.yml

```diff
@@ -43,3 +43,39 @@ docker_builder:
     docker buildx build --platform linux/amd64,linux/arm64 --push \
       --tag ghcr.io/cirruslabs/android-sdk:34-ndk \
       sdk/34-ndk
+
+task:
+  name: Smoke Test
+  container:
+    dockerfile: sdk/34/Dockerfile
```

Comment:
We should probably depend on the `sdk/tools/Dockerfile` here and add sdkmanager installation to `setup_script`. Otherwise changes to `sdk/tools/Dockerfile` won't be actually smoke tested.

Reply:
Ah, this makes sense.